babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ['@babel/preset-env', '@babel/preset-typescript', ['@babel/preset-react', { runtime: 'automatic' }]],
```

Comment:
same comments regarding babel from the previous PR. I would check that the bundle is the same as before. also publish alpha for you/Pavel after it

Reply:
Pavel did a sanity

Reply:
if the bundle is the same as before you can merge

Reply:
How do you compare the bundle?